doc/Language/101-basics.pod6

```diff
@@ -74,37 +66,55 @@ for @sorted -> $n {
 
 This produces the output:
 
-=for code :lang<text>
+=begin code :lang<text>
+Tournament Results:
+
 Ana has won 2 matches and 8 sets
 Dave has won 2 matches and 6 sets
 Charlie has won 1 match and 4 sets
 Beth has won 1 match and 4 sets
+=end code
 
-=head3 X<C<v6>|v6 (Basics)>
+=head1 Pragma X<C<v6>|v6 (Basics)>
+
+=begin code :preamble<use v6>
```

Comment:
Er. You add the preamble to the code itself? Wouldn't this be :solo?

Reply:
I guess this configuration information is special to Documentable. Is there some documentation on how it uses them?

Reply:
Well, the preamble thing is common to all documentation. What I'm saying is that it makes no sense to repeat the code in the preamble. And that in these case, IARA, you can use :solo so that it's compiled in a single file, although I guess it's OK if you simply elminate the preamble.